table/block_based_table_reader.h

```diff
@@ -59,9 +59,16 @@ class GetContext;
 
 typedef std::vector<std::pair<std::string, std::string>> KVPairBlock;
 
-// A Table is a sorted map from strings to strings.  Tables are
-// immutable and persistent.  A Table may be safely accessed from
-// multiple threads without external synchronization.
+// Reader class for BlockBasedTable format.
+// For the format of BlockBasedTables refer to https://fburl.com/xc8lqvwy.
```

Comment:
BlockBasedTables->BlockBasedTable?

Reply:
Also I am not sure if external developer can access fburl.

Reply:
pretty sure they cannot